kf_lib_data_ingest/etl/load/load.py

```diff
@@ -534,3 +553,9 @@ def _run(self, target_entities):
                 f"⚠️ Could not find resume_from target '{self.resume_from}'! "
                 "Nothing was actually loaded into the target service."
             )
+
+        json_out = self.msg_out.replace(".txt", ".json")
```

Comment:
What happens if there is an exception in `_run` before writing out. Maybe we want something like:

```
def _run(blah):
    try:
        do _run stuff
    except Exception as e:
        write output
        raise e
    else:
        write output
```

Reply:
only if you want it to look like it completed

Reply:
I think we do. In fact, I think I'd look at that file especially if there was some sort of exception just to see what got sent before it failed